openapi/paths/quotes@{id}@recall.yaml

```
@@ -0,0 +1,26 @@
+parameters:
+  - $ref: ../components/parameters/resourceId.yaml
+post:
+  x-products:
+    - Core
+  tags:
+    - Quotes
+  summary: Recall a quote
+  operationId: PostQuoteRecall
+  x-sdk-operation-name: recall
+  description: Recalls an issued quote with a specified ID. After recall the quote status is `draft`.
```

nitpick: I think we should use `retract` here

`Recall` is ambiguous because it is often used in relation to our memory. For example, "I can still vaguely recall being taken to the hospital".

```suggestion
  summary: Retract a quote
  operationId: PostQuoteRecall
  x-sdk-operation-name: recall
  description: Retract an issued quote with a specified ID. Retracted quotes are assigned the `draft` status.
```